src/components/DateRangePicker.jsx

```diff
@@ -108,7 +109,7 @@ export default class DateRangePicker extends React.Component {
         endDate = null;
       }
     } else if (focusedInput === END_DATE) {
-      if (isInclusivelyBeforeDay(day, startDate)) {
+      if (minimumNights > 0 && isInclusivelyBeforeDay(day, startDate)) {
```

Comment:
Won't this allow you to select end dates before the start date, if minimumNights is set to zero?